htdocs/mrp/mo_agenda.php

```diff
@@ -209,7 +209,7 @@
 		if (get_class($objthirdparty) == 'Societe') {
 			$out .= '&amp;socid='.$objthirdparty->id;
 		}
-		$out .= (!empty($objcon->id) ? '&amp;contactid='.$objcon->id : '').'&amp;backtopage=1&amp;percentage=-1';
+		$out .= (!empty($objcon->id) ? '&amp;contactid='.$objcon->id : '').'&amp;backtopage='.$_SERVER["PHP_SELF"].'?id='.$object->id.'&amp;percentage=-1';
```

Comment:
You must add urlencode() around the string $_SERVER["PHP_SELF"].'?id='.$object->id

Reply:
Is done. Many things to learn in the PHP and web world... :)